### ISSUE TITLE
Question about training details.

### ISSUE THREAD

**Issue by mls1999725** [YadiraF/PIXIE#42]
Hi, I'm very interested in your excellect work. Could you please help me with a few questions that I didn't find the answer to in the paper?
1. In the first stage of training, what are the hyperparameters (learning rate and weight decay of the optimizer and number of training epochs) for body, hand and face? 
2. The prior mentioned in the paper is added on which loss in which stage of training? 
3. What is the loss weight of body, hand, face and update loss?
Thanks very much!
